sdk/azcore/arm/poller.go

```diff
@@ -0,0 +1,83 @@
+//go:build go1.16
```

Comment:
Should this be part of the pollers package?

Reply:
The creation of ARM pollers has been moved to package `azcore/arm/runtime` to mimic what we do in core.